tests/test_api.py

```diff
@@ -154,9 +154,34 @@ class MockParser(object):
                                    TEST_GIT_BRANCH, TEST_USER,
                                    TEST_COMPONENT, TEST_TARGET, TEST_ARCH)
 
+
+    @pytest.mark.parametrize('label_name', ['BZComponent', 'com.redhat.component', 'Name', 'name'])
+    def test_missing_component_and_name_labels(self, osbs, label_name):
```

Comment:
Does this make the `test_create_prod_build_missing_name_label` unit test above it obsolete?